data/docs/rules-v2.md

```diff
@@ -0,0 +1,204 @@
+title: Comment jouer ?
+
+Dans le Corporate Game, vous incarnez une pointure dans l’univers de la finance dans un futur où le capitalisme a fini par dissoudre les pays. Vous jouez à Detroit, la première ville libérée du contrôle des États. Bienvenue dans le Disneyland des traders !
+
+Ces règles vont détailler toutes les actions que vous avez la possibilité de faire à chaque tour. Le jeu se joue en 7 tours de 24h. À 13h chaque jour, les actions de chaque joueur seront résolues en même temps et un nouveau tour pourra commencer. La personnalité la plus influente de Detroit sera sacrée à la fin du 7<sup>eme</sup> tour, les autres sombreront dans l’oubli.
+
+[TOC]
+
+## Le classement corporatiste
+> Le classement de fin d’année c’est un peu comme une course. Tu peux avoir été devant tout le long, si le dernier te passe devant juste avant la ligne, c’est sa sale tronche qui sera dans le journal. 
+
+Votre objectif est de devenir la personnalité la plus respectée de Detroit à la fin de l’année. Le truc important à retenir, c’est qu’il n’y a pas de victoire en équipe. Il y aura celui qui a réussi et ceux qui ont échoué. Et commencez pas à parler de chance, personne n’aime les gens qui se plaignent !
+ 
+> Pourquoi je ne mise que sur 10 corpos parmi des centaines ? Tu m’as bien regardé ? J’ai l’air d’un hippie qui laisse traîner son fric dans une boite réglo ?
+
+Votre champ de bataille, c’est 10 énormes entreprises cotées à Détroit. On les appelle des corporations, ou **corpos**. Chaque corpo commence avec un nombre d’**actifs** qui va évoluer au cours de la partie, au départ les corpos commencent avec entre 8 et 13 actifs.
```

Comment:
Je pense que tu devrais t'attarder sur le concept d'actif. Le fait que c'est ce qui représente la taille de la corpo and such.

Pour les actifs de départs, pour l'instant le suis à 3x8, 4x10 et 3x12, mais ça se change si tu veux.

Aussi, je mentionnerais le classement avant l'achat de part, vu que t'as besoin de savoir ce qu'il y a une première corpo.